Source/Fuse.Reactive.JavaScript/Tests/UX/ClassConstructor.ux

```diff
@@ -1,6 +1,5 @@
 <UX.ClassConstructorBase ux:Class="UX.ClassConstructor">
 	<Panel ux:Dependency="A" ux:Binding="A" />
 	<Panel ux:Dependency="B" /> <!--  This one is here to force this and UX.ClassConstructorBase to get different constructor signatures -->
-	<Panel ux:Name="_a" Children="A" ux:AutoBind="false" />
```

Comment:
You can't do this. This is a regression-test that verifies that we don't generate invalid Uno-code in this case, changing this completely undermines the test-case.